pom.xml

```diff
@@ -2023,6 +2032,9 @@
                             <include>src/test/java/**</include>
                             <include>src/test/scala/**</include>
                         </includes>
+                        <excludes>
+                            <exclude>**/*.txt</exclude>
```

Comment:
what text files do we have in src/main/java (scala) ? any text files should go in src/main/resources